src/api/EntitySetsApi.js

```diff
@@ -387,6 +392,53 @@ function getPropertyTypeMetaDataForEntitySets(entitySetIds :UUID[]) :Promise<*>
     });
 }
 
+/**
+ * `PATCH /entity-sets/all/{id}/metadata`
+ *
+ * Updates the metadata for a given entity set.
+ *
+ * @static
+ * @memberof lattice.EntitySetsApi
+ * @param {UUID[]} entitySetId ID for entity set.
+ * @param {Object} update Only title, description, contacts and name fields are accepted. Other fields are ignored.
+ * @returns {Promise<void>} - a Promise that resolves without a value
+ *
+ * @example
+ * EntityDataModelApi.updateEntitySetMetadata(
+ *   "0c8be4b7-0bd5-4dd1-a623-da78871c9d0e",
+ *   {
+ *     "title": 'titleString',
+ *     "description": 'descriptionString',
+ *     "contacts": ["contact_1", "contact_2"],
+ *     "name": 'nameString'
+ *   }
+ * )
+ */
+function updateEntitySetMetadata(entitySetId :UUID, update :Object) :Promise<number> {
```

Comment:
I know metadata is one word, so `Metadata` is correct. But since we use `MetaData` in other places, let's stick with it here to be consistent:

https://github.com/openlattice/lattice-js/blob/develop/src/api/EntityDataModelApi.js#L910
https://github.com/openlattice/lattice-js/blob/develop/src/api/EntityDataModelApi.js#L990
https://github.com/openlattice/lattice-js/blob/develop/src/api/EntitySetsApi.js#L322
https://github.com/openlattice/lattice-js/blob/develop/src/api/EntitySetsApi.js#L370

Reply:
I don't like it, but I'll do it :)